src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const firstNumber = Big(numberOne);
+  const secondNumber = Big(numberTwo);
+  let result = 0;
+
+  if (operation === '+') {
+    result = firstNumber.plus(secondNumber);
+  } else if (operation === '-') {
+    result = firstNumber.minus(secondNumber);
+  } else if (operation === 'X') {
+    result = firstNumber.times(secondNumber);
+  } else if (operation === '÷') {
+    result = firstNumber.div(secondNumber);
+  } else if (operation === 100) {
```

Comment:
- [ ] The operation here should be `%`, not `100` since `100` isn't a calculator operation.